task-library/AHV.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+##############################################
+# Name        : AHV.py
+# Author      : Calm Devops
+# Version     : 1.0
+# Description : Script will list all VMs available in a PC and update a VM
+##############################################
+import requests
+import json
+import re
+import time
+import sys
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+class RestUtilException(Exception):
+    pass
+
+def _wrap(resp):
```

Comment:
Please add doc_string which will detailed about function purpose, in args, return value etc..